spacy/lang/la/lex_attrs.py

```diff
@@ -0,0 +1,32 @@
+from ...attrs import LIKE_NUM
+import re
+
+# cf. Goyvaerts/Levithan 2009
+roman_numerals_compile = re.compile(r'(?i)^(?=[MDCLXVI])M*D?C{0,4}L?X{0,4}V?I{0,4}$')
```

Comment:
I've been testing this one a bit more, and it looks like `IV` and `iv` aren't covered by this regular expression, same issue for `ix` and `IX`.